api/app/GraphQL/Mutations/DisqualifyCandidate.php

```diff
@@ -22,6 +22,10 @@ public function __invoke($_, array $args)
         $candidate->final_decision_at = $now;
         $candidate->save();
 
+        $finalDecicion = $candidate->computeFinalDecision();
+        $candidate->computed_final_decision = $finalDecicion['decision'];
+        $candidate->computed_final_decision_weight = $finalDecicion['weight'];
```

Comment:
Oops there's also a typo here. Is should be `$finalDecision`.

You'll need to fix that in the other files as well:

- [ ] PlaceCandidate.php
- [ ] QualifyCandidate.php
- [ ] RenistateCandidate.php
- [ ] RemoveCandidate.php
- [ ] RevertFinalDecision.php
- [ ] RevertPlaceCandidate.php
- [ ] SubmitApplication.php

Reply:
Fixed in [fix typo](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10920/commits/2dea74fbe471ea8538bc6c5152faa655d710c48f)